templates/_base/worker/files/container-registries.yaml

```diff
@@ -0,0 +1,30 @@
+filesystem: "root"
```

Comment:
It DRY things up a bit to have a `templates/_base/all/files/container-registries.yaml` or some such which would be added to both masters and workers.  But perhaps that is more than we want to bite off in this particular PR.